<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_LED1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LED2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LD2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LED3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LED4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_restart">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shutdown">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>shutdown</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(5, 0, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mount">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 40);</string>
    </property>
    <property name="text">
     <string>mount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_umount">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 246);</string>
    </property>
    <property name="text">
     <string>umount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 249, 164);</string>
    </property>
    <property name="text">
     <string>&gt;&gt; Already started sucessful</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_serial_connct">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_serial_disconnect">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Enable">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor ON</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Disable">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J1F">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J1+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J1B">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J1-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J2F">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J2+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J2B">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J2-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J3F">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J3+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J3B">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J3-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J4F">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J4+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_J4B">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J4-</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1310720</number>
    </property>
    <property name="singleStep">
     <number>1000</number>
    </property>
    <property name="pageStep">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
